.vs/ProjectSettings.json

```diff
@@ -0,0 +1,3 @@
+{
+  "CurrentProjectSetting": null
+}
```

Comment:
Are we aware of why these `vs` files have been altered/added?

Reply:
Thank you for catching that, I don't know how those were added in.

Reply:
No problem, there are all kinds of settings going on, hard to keep track of sometimes.